utils/date.go

```diff
@@ -132,6 +132,7 @@ func IsoWeekToDate(year, week int, weekday time.Weekday) (time.Time, error) {
 }
 
 // MostRecentMonday returns the most recent Monday before d.
+// The resulting date has the same time as d.
```

Comment:
Added this comment because this little gotcha is why we need to do `StartOfDayOn` in `FormatWeekRelativeTo`.